pkg/engine/api/imageverifymetadata.go

```diff
@@ -29,11 +37,22 @@ func (ivm *ImageVerificationMetadata) IsVerified(image string) bool {
 	if !ok {
 		return false
 	}
+	return verified == ImageVerificationPass || verified == ImageVerificationSkip
+}
+
+func (ivm *ImageVerificationMetadata) ImageVerificationStatus(image string) ImageVerificationMetadataStatus {
+	if ivm.Data == nil {
+		return ImageVerificationFail
```

Comment:
Is this considered Fail or Error status? For rule status we define:
```
	// RuleStatusError indicates that the policy rule could not be evaluated due to a processing error, for
	// example when a variable cannot be resolved  in the policy rule definition. Note that variables
	// that cannot be resolved in preconditions are replaced with empty values to allow existence
	// checks.
```
```
	// RuleStatusFail indicates that the resource does not meet the policy rule requirements
```

Reply:
@realshuting , `ImageVerificationFail ` is not a rule `fail`, it is just the status of image ref in the verify-images annotation, the only values there are `pass`, `fail` and `skip`. I have merged `fail` and `error` because in this case there are no advantages (IMO) to handling them separately